docs/source/using/image-caching.adoc

```diff
@@ -4,39 +4,150 @@ include::variables.adoc[]
 :icons:
 :toc: macro
 :toc-title:
-:toclevels: 1
+:toclevels: 2
 
 toc::[]
 
 [[image-caching-overview]]
 == Overview
 
-To speed up provisioning of the OpenShift cluster and to minimize network traffic, the core OpenShift images can be cached on the host.
-This feature is considered experimental and needs to be explicitly enabled.
+To speed up the provisioning of the OpenShift cluster and to minimize network traffic, container images can be cached on the host.
+These images can then be imported into the running Docker daemon, either explicitly on request or implicitly during xref:../using/basic-usage.adoc#minishift-start-overview[`minishift start`].
+The following sections describe image caching and its configuration in more detail.
 
-[[image-chaching-configuration]]
-== Image Caching Configuration
+[TIP]
+====
+The format in which images are cached has changed with {project} version 1.10.0.
+Prior to 1.10.0 the images were stored as tar files.
```

Comment:
Were they stored as tar files or as .tar.gz compressed archives? (I'm honestly not sure and just want to ensure that we're being accurate.)

Reply:
tar files